decidim-elections/decidim-elections.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "LICENSE-AGPLv3.txt", "Rakefile", "README.md"]
 
-  s.add_dependency "decidim-bulletin_board", "0.16.1"
+  s.add_dependency "decidim-bulletin_board", "0.17.0"
 
   # Note: these gems will be moved to the application in the next release
-  s.add_dependency "voting_schemes-dummy", "0.16.1"
-  s.add_dependency "voting_schemes-electionguard", "0.16.1"
+  s.add_dependency "voting_schemes-dummy", "0.17.0"
+  s.add_dependency "voting_schemes-electionguard", "0.17.0"
```

Comment:
I already released `0.17.1` 🙃. Even if it's not critical update it if possible. Also when updating these gems we need to update the workflows as well to use the correct server image. Just replace the `0.16.0` with `0.17.1` as well.